bgpd/bgp_vty.c

```diff
@@ -533,6 +533,50 @@ int bgp_vty_find_and_parse_afi_safi_bgp(struct vty *vty,
 	return *idx;
 }
 
+int bgp_vty_find_and_parse_bgp(struct vty *vty,
```

Comment:
why is `bgp_vty_find_and_parse_afi_safi_bgp` insufficient here?

Reply:
you are right.
I will use afi_safi_bgp